docs/utils.md

```diff
@@ -54,3 +54,63 @@ Timeframes available for the exchange `binance`: 1m, 3m, 5m, 15m, 30m, 1h, 2h, 4
 ```
 $ for i in `freqtrade list-exchanges -1`; do freqtrade list-timeframes --exchange $i; done
 ```
+
+## List pairs/list markets
+
+The `list-pairs` and `list-markets` subcommands allow to see the pairs/markets available on exchange.
+
+Pairs are markets with the '/' character between the base currency part and the quote currency part.
```

Comment:
Maybe also add a quick sample on which part is quote / and which is base? 
`<BASE>/<QUOTE>`
or "ETH/BTC - where ETH is the base currency, and BTC is the quote currency."... ?

Reply:
Added, pls review.